modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/css/pages/_form_view.scss

```diff
@@ -81,7 +81,9 @@
 
 			h3.category-text {
 				color: $grayColor700;
+				font-size: 12px;
```

Comment:
Hey @rebsilva , looks strange for me, to decrease the font-size of h3 to 12px.

I think, would be better change this H3 to Span or the Small Tag, and use their default values, you can look at this W3C school snippet 

https://www.w3schools.com/cssref/tryit.asp?filename=trycss_default_small

Reply:
That's true Keven, i already did the modification. Thanks!